packages/expo-file-system/ios/Next/FileSystemNextExceptions.swift

```diff
@@ -12,3 +14,9 @@ internal class MoveDirectoryToFileException: Exception {
     "Unable to move a directory to a file"
   }
 }
+
+internal class UnableToDownloadException: GenericException<String> {
```

Comment:
Is this OK, or should we make several reasons (no response, invalid response code (300/400), other)?